uktrade/data-hub-frontend
# Added accesibility statement page and updated footer links

## Description of change

This PR adds an accesibility statement page and updated footer links to _Export wins review_.

## Test instructions

1. Go to `/exportwins/review/<token>`
2. In the footer...
3. Click on the _Privacy policy_ link
4. It should take you to `https://www.great.gov.uk/privacy-and-cookies/full-privacy-notice/` in a new tab
5. Click on the _Accessibility Statement_
6. It should take you to `/exportwins/review/accesibility-statement` in a new tab
